openshift/catalog-sync/test/suite_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github/operator-framework-operator-controller/openshift/catalog-sync/pkg/check"
+	"github/operator-framework-operator-controller/openshift/catalog-sync/pkg/extract"
+)
+
+func TestSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Catalog Validation Suite")
+}
+
+// images contains the list of images to be validated.
+var images = []string{
+	"registry.redhat.io/redhat/community-operator-index:v4.18",
+	"registry.redhat.io/redhat/redhat-marketplace-index:v4.18",
+	"registry.redhat.io/redhat/certified-operator-index:v4.18",
+	"registry.redhat.io/redhat/redhat-operator-index:v4.18",
+}
```

Comment:
For now it is okay for these to be hardcoded, but let's change these to 4.19 tags. We are technically now in 4.20 development window, but I don't think 4.20 catalogs have been bootstrapped.

As soon as this merges, we should open a PR to bump these to 4.20 (which will fail), and then we go hound the pipelines to get the 4.20 bootstrapping done.

As a follow-up to this PR, we should avoid maintaining this list separately. We already have a set of default catalogs in our openshift manifests, so it would be nice to derive this list from our existing default `ClusterCatalog` objects.

Reply:
OK. I will update it and give a look